gps_ubxm8_i2c/ubxMessages.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <stdint.h>
+
+enum class GPSFixType : uint8_t {
+    NO_FIX,
+    DEAD_RECKONING,
+    FIX_2D,
+    FIX_3D,
+    GNSS_AND_DEAD_RECKONING,
+    TIME_ONLY_FIX
+};
+
+static uint8_t PVT_MESSAGE[4] = {0x01, 0x07, 0x00, 0x00};
+
+// https://content.u-blox.com/sites/default/files/products/documents/u-blox8-M8_ReceiverDescrProtSpec_UBX-13003221.pdf#%5B%7B%22num%22%3A777%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C0%2C654.8%2Cnull%5D
```

Comment:
clean up url, just `https://content.u-blox.com/sites/default/files/products/documents/u-blox8-M8_ReceiverDescrProtSpec_UBX-13003221.pdf` is a valid link